### ISSUE TITLE
Coverage incorrectly reported as 0 for C# project

### ISSUE THREAD

**Issue by julianghionoiu** [julianghionoiu/dpnt-coverage#31]
For this user, the coverage failed to compute correctly:
https://github.com/DPNT-Sourcecode/CHK-qjno01

Needs some investigation in regards to what went wrong.

**Comment by neomatrix369** [comment]
PR for fix at https://github.com/julianghionoiu/tdl-runner-csharp/pull/11

**Comment by neomatrix369** [comment]
Ran code at https://github.com/DPNT-Sourcecode/CHK-qjno01 after applying fix julianghionoiu/tdl-runner-csharp#11:

```
+ [[ unused != http* ]]
+ echo 'SQS_QUEUE_URL does not seem to be valid. Will print to the console and exit'
+ echo 'participant=qjno01 roundId=R2 coverage=99'
participant=qjno01 roundId=R2 coverage=99
+ exit 0
```

Issue caused due to not finding `nunit3-console.exe` in the project due to the participant upgrading the coverage tool.

**Comment by neomatrix369** [comment]
Fix applied to FSharp and VBNet projects as well
